package.json

```diff
@@ -38,18 +38,23 @@
   ],
   "sideEffects": false,
   "main": "./src/index.ts",
+  "legacy": "./legacy/src/index.js",
   "types": "./dist/index.d.ts",
   "publishConfig": {
     "main": "./dist/index.cjs",
     "module": "./dist/index.js",
+    "legacyCjs": "./legacy/index.cjs",
+    "legacyModule": "./legacy/index.js",
```

Comment:
it's not supported fields. Lets move this info into legacy/package.json

Reply:
moved